Open Call Champions Program 2026 (#1218)
ropensci/roweb3: content/blog/2026-01-12-opencall-champions-program/index.en.md

---
title: Open call for the rOpenSci Champion Program 2026!
author:
- Yanina Bellini Saibene
editor:
- Steffi LaZerte
date: '2026-01-12'
slug: programchamps2026
featured: true
tags:
- champions program

> description: You can now apply to participate in the 2026 Champions program. We

Reviewer: ```suggestion
description: You can now apply to participate in the 2026 Champions Program. We
```

  are looking for people to participate in mentoring roles and to become Champions.
params:
  doi: '10.59350/1mmc4-bqp35'
---

> We are pleased to announce the opening of a new call for applications for the **rOpenSci Champions Program in Spanish**, which will begin in 2026. We will be accepting applications beginning in **January 12, 2026** and until **February 20, 2026** for both the role of **Champion** as well as for **Mentor**.

Reviewer: ```suggestion
We are pleased to announce the opening of a new call for applications for the **rOpenSci Champions Program in Spanish**, which will begin in 2026. We will be accepting applications beginning in **January 12, 2026** and until **February 20, 2026** for both the roles of **Champion** as well as **Mentor**.
```

> As in the previous cohort, the program will be developed **entirely in Spanish.** and will have a **regional focus on Latin America** with the objective of further strengthening the research and open science software in the region.

Reviewer: ```suggestion
As in the previous cohort, the 2026 program will be developed **entirely in Spanish** and will have a **regional focus on Latin America** with the objective of further strengthening the research and open science software in this region.
```

## What is the Champions Program?

> The Program seeks to identify, accompany and recognize people who are already leading, or who want to take a step further, in building open science and sustainable research software communities.

Reviewer: I think 'support' is a bit better here? 'Accompany' implies 'going along with', not necessarily helping or supporting. 

```suggestion
This program seeks to identify, support and recognize people who are already leading, or who want to take a step further, in building open science and sustainable research software communities.
```

Throughout **12 months** the selected individuals will participate in:

- An initial training of 13 2-hour workshops cohorts based.

> - The development of a research software project: creation of an R package, participation in the rOpenSci peer review with an authored package or as part of a team of reviewers.

Reviewer: ```suggestion
- An initial cohort-based training of 13 2-hour workshops.
- The development of a research software project: creation of a new R package, submission of an existing R package to rOpenSci peer review, or as part of a team of reviewers of other R packages.
```

- Use R and have previous experience in software development, reproducible analysis or open science.
- Are interested in developing an open research software project and strengthening their local or regional communities.
- Can communicate effectively in Spanish (oral and written).
- Accept our [Code of Conduct](/code-of-conduct/) and be able to commit to participate in the program activities organized for champions during these 12 months.

### Mentor(a|e)s

We invite applications as mentors from people who:

- Have experience developing, maintaining or contributing to R packages (ideally within the rOpenSci ecosystem, although this is not exclusive).
- Have participated in software peer review processes.
- Have relevant mentoring experience.
- Have an interest in mentoring others, sharing experiences and actively contributing to the growth of the program.
- Be able to communicate effectively in Spanish (oral and written).
- Accept our [Code of Conduct](/code-of-conduct/) and be able to commit to participate in the program activities organized for mentors during these 12 months.

## Key dates of the 2026 call

- **Opening of the call:** Monday, January 12, 2026

> - [**Conversation with the information community:** Wednesday, January 21, 2026](/es/commcalls/champions-latino-2025/)

Reviewer: I'm confused why these link to a previous event. Perhaps create the event they should link to first? Then show this link below in the description (see my suggestion there)
```suggestion
- [**Community Call - Conversation with the information community:** Wednesday, January 21, 2026](/es/commcalls/champions-latino-2025/)
```

- [**Application Clinics:** Thursday, February 5, 2026](/events/clinica-champions-2026/)
- **Closing date:** Friday, February 20, 2026

The *Community Call* on January 21 will feature the participation of **champions and mentors from previous cohorts.** They will share their experiences and answer questions about the program, and we invite you to join us!

During February we will hold one **application clinic**. It is an open space where you will be able to receive help to complete the application form, resolve doubts and receive direct guidance from the program team.

## How to apply?

Applications will be made through online forms:

- [Complete this form to apply to become a Champion(a|e)s](https://airtable.com/appF6OXmxkk8VmR8a/shrlvgcNz1R4CoaQN)

> - [Complete this form to apply to become a Mentor(a|e)s)](https://airtable.com/appF6OXmxkk8VmR8a/shrZnWDRKxXUEmLRM),

Reviewer: ```suggestion
- [Complete this form to apply to become a Champion](https://airtable.com/appF6OXmxkk8VmR8a/shrlvgcNz1R4CoaQN)
- [Complete this form to apply to become a Mentor](https://airtable.com/appF6OXmxkk8VmR8a/shrZnWDRKxXUEmLRM),
```

These forms are also available at [program website](https://champions-program.ropensci.org).

> The forms must be completed **in Spanish**. On the program's website you will also find [the details of the information required to apply as a champion.](https://champions-program.ropensci.org/champions/#información-requerida-en-el-formulario) or [as a mentor](https://champions-program.ropensci.org/mentors/#información-requerida-en-el-formulario) to facilitate the preparation of applications and [answers to frequently asked questions](https://champions-program.ropensci.org/resources/faq.html).

Reviewer: ```suggestion
The forms must be completed **in Spanish**. On the program's website you will also find more details regarding the requirements for [Champions](https://champions-program.ropensci.org/champions/#información-requerida-en-el-formulario) and [Mentors](https://champions-program.ropensci.org/mentors/#información-requerida-en-el-formulario), as well as [answers to frequently asked questions](https://champions-program.ropensci.org/resources/faq.html).
```

> ## Why participate?

Reviewer: Consider putting this section before "How to apply?"
